src/Utils/Path.idr

```diff
@@ -0,0 +1,437 @@
+module Utils.Path
+
+import Data.List
+import Data.Maybe
+import Data.Strings
+import Data.String.Extra
+import System.Info
+import Text.Token
+import Text.Lexer
+import Text.Parser
+import Text.Quantity
+
+infixr 5 </>
+infixr 7 <.>
+
+
+isWindows : Bool
+isWindows = os `elem` ["windows", "mingw32", "cygwin32"]

```

Comment:
We should probably move this to System.Info, it exists in Core.Options too.

Reply:
Yes! I'm still working in the process. Maybe I should change the title a bit...